readme.md

```diff
@@ -52,12 +52,14 @@ Length of the returned string.
 
 Type: `string`<br>
 Default: `'hex'`<br>
-Values: `'hex'` `'base64'` `'url-safe'`
+Values: `'hex'` `'base64'` `'url-safe'` `'distinguishable'`
 
 Use only characters from a predefined set of allowed characters.
 
 Cannot be set at the same time as the `characters` option.
 
+The `distinguishable` set contains only uppercase characters that are not easily confused: `CDEHKMPRTUWXY012458`.
```

Comment:
This needs to be added to index.d.ts too. Readme and index.d.ts should be in sync.